src/main/java/capstone/examlab/questions/controller/QuestionsController.java

```diff
@@ -22,43 +25,50 @@
 @Slf4j
 @RequiredArgsConstructor
 @RestController
-@RequestMapping("api/v1/exams/")
+@RequestMapping("api/v1/")
 public class QuestionsController {
     private final QuestionsService questionsService;
 
     //Create API
-    @PostMapping("{examId}/questions")
-    public ResponseEntity<String> addQuestionsByExamId(@PathVariable @ValidExamId Long examId, @RequestPart QuestionUploadInfo questionUploadInfo, @RequestPart(name = "questionImagesIn", required = false) List<MultipartFile> questionImagesIn,
-                                                       @RequestPart(name = "questionImagesOut", required = false) List<MultipartFile> questionImagesOut, @RequestPart(name = "commentaryImagesIn", required = false) List<MultipartFile> commentaryImagesIn, @RequestPart(name = "commentaryImagesOut", required = false) List<MultipartFile> commentaryImagesOut) {
-        boolean saved = questionsService.addQuestionsByExamId(examId, questionUploadInfo, questionImagesIn, questionImagesOut, commentaryImagesIn, commentaryImagesOut);
-        if (saved) {
-            return ResponseEntity.ok("question add suucess");
-        } else {
-            return ResponseEntity.badRequest().body("question add error");
+    @ResponseStatus(HttpStatus.CREATED)
+    @PostMapping("exams/{examId}/questions")
```

Comment:
```suggestion
    @PostMapping("/exams/{examId}/questions")
```
위 제안처럼 변경된다면 이렇게 바뀌는게 좋겠습니다

Reply:
그러네요 실수한 부분 변경할게요